root/types.js

```diff
@@ -926,9 +926,8 @@ declare type SanitizedCatalystContextT = {
   +action: {
     +name: string,
   },
-  +req: {
```

Comment:
All the changes outside of root/static/scripts/common/components/Annotation.js should probably be removed now BTW.